pyvista/core/filters/data_set.py

```diff
@@ -4613,6 +4613,76 @@ def shrink(self, shrink_factor=1.0, progress_bar=False):
             return output.extract_surface()
         return output
 
+    def tessellate(self, max_n_subdivide=3, merge_points=True, progress_bar=False):
+        """Tessellate a mesh.
+
+        This filter approximates nonlinear FEM-like elements with linear
+        simplices. The output mesh will have geometry and any fields specified
+        as attributes in the input mesh's point data. The attribute's copy
+        flags are honored, except for normals.
+
+        For more details see `vtkTessellatorFilter <https://vtk.org/doc/nightly/html/classvtkTessellatorFilter.html#details>`_.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Parameters
+        ----------
+        max_subdivisions : int, optional
```

Comment:
```suggestion
        max_n_subdivide : int, optional
```

Reply:
There's a similar kwarg in `extract_surface()`: https://github.com/pyvista/pyvista/blob/47e68c1d94ee0c70f0b562ba52c96cc11e531561/pyvista/core/filters/data_set.py#L4055-L4068

Not exactly the same functionality, just FYI looking at this kwarg being renamed.